rest/admin_api_test.go

```diff
@@ -569,8 +567,7 @@ func TestLoggingCombined(t *testing.T) {
 }
 
 func TestGetStatus(t *testing.T) {
-	rtConfig := RestTesterConfig{NoFlush: true}
-	rt := NewRestTester(t, &rtConfig)
+	rt := NewRestTester(t, nil)
```

Comment:
I'm assuming from this that there isn't significant value in avoiding the new (query-based) flush when we know it's not necessary?

Reply:
This option won't do anything once the bucket pooling stuff goes in, as buckets will be flushed before the test gets a reference to it.

I'm happy to remove from this PR until that happens, or we can just keep the temporary overhead of flushing in these tests in the meantime?